<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab_Sys</class>
 <widget class="QWidget" name="Tab_Sys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="edit_qsub">
        <property name="toolTip">
         <string>Command used to submit jobs to the PBS queue. Usually qsub.</string>
        </property>
        <property name="text">
         <string>qsub</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_launch">
        <property name="toolTip">
         <string>Command used to submit jobs to the PBS queue. Usually qsub.</string>
        </property>
        <property name="text">
         <string>Queue submit program:</string>
        </property>
        <property name="buddy">
         <cstring>edit_qsub</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="5">
       <widget class="QGroupBox" name="cb_remote">
        <property name="title">
         <string>&amp;Remote server settings</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Host:</string>
           </property>
           <property name="buddy">
            <cstring>edit_host</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>User:</string>
           </property>
           <property name="buddy">
            <cstring>edit_username</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="edit_host"/>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="edit_username"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Path:</string>
           </property>
           <property name="buddy">
            <cstring>edit_rempath</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QLineEdit" name="edit_rempath"/>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="spin_port">
           <property name="prefix">
            <string>SSH Port </string>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
           <property name="value">
            <number>22</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="edit_qstat">
        <property name="toolTip">
         <string>Command used to check the PBS queue. Use qstat.</string>
        </property>
        <property name="text">
         <string>qstat</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_path">
        <property name="toolTip">
         <string>Local path where files are to be written.</string>
        </property>
        <property name="text">
         <string>Path:</string>
        </property>
        <property name="buddy">
         <cstring>edit_path</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="edit_path">
        <property name="toolTip">
         <string>Local path where files are to be written.</string>
        </property>
        <property name="text">
         <string>/tmp</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_check">
        <property name="toolTip">
         <string>Command used to check the PBS queue. Use qstat.</string>
        </property>
        <property name="text">
         <string>Queue checking program:</string>
        </property>
        <property name="buddy">
         <cstring>edit_qstat</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="edit_description">
        <property name="toolTip">
         <string>Base name of individual structure files.

Final name will be: 
[base name][generation number]x[id number]</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_description">
        <property name="toolTip">
         <string>Base name of individual structure files.

Final name will be: 
[base name][generation number]x[id number]</string>
        </property>
        <property name="text">
         <string>Optimization description:</string>
        </property>
        <property name="buddy">
         <cstring>edit_description</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_qdel">
        <property name="toolTip">
         <string>Command used to delete jobs from the queue. Usually qdel.</string>
        </property>
        <property name="text">
         <string>Job delete command:</string>
        </property>
        <property name="buddy">
         <cstring>edit_qdel</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="edit_qdel">
        <property name="toolTip">
         <string>Command used to delete jobs from the queue. Usually qdel.</string>
        </property>
        <property name="text">
         <string>qdel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edit_path</tabstop>
  <tabstop>edit_description</tabstop>
  <tabstop>edit_qsub</tabstop>
  <tabstop>edit_qstat</tabstop>
  <tabstop>edit_qdel</tabstop>
  <tabstop>cb_remote</tabstop>
  <tabstop>edit_host</tabstop>
  <tabstop>edit_username</tabstop>
  <tabstop>edit_rempath</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
